packages/rest/src/coercion/utils.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as debugModule from 'debug';
+
+const debug = debugModule('loopback:rest:coercion');
+
+export function isEmpty(data: string) {
+  debug('isEmpty %s', data);
+  return data === '';
+}
+/**
+ * A set of truthy values. A data in this set will be coerced to `true`.
+ *
+ * @param data The raw data get from http request
+ * @returns The corresponding coerced boolean type
+ */
+export function isTrue(data: string): boolean {
+  return ['TRUE', '1'].includes(data.toUpperCase());
+}
+
+/**
+ * A set of falsy values. A data in this set will be coerced to `false`.
+ * @param data The raw data get from http request
+ * @returns The corresponding coerced boolean type
+ */
+export function isFalse(data: string): boolean {
+  return ['FALSE', '0'].includes(data.toUpperCase());
+}
+
+export function isSafeInteger(data: number) {
+  return -Number.MAX_SAFE_INTEGER <= data && data <= Number.MAX_SAFE_INTEGER;
+}
+
+export function isValidDateTime(data: Date) {
+  return isNaN(data.getTime()) ? false : true;
```

Comment:
I think a more accurate name for this function is `isValidDate`

Reply:
👍 i will try `isValidDate`, actually, I am looking for some community validator for ["Internet Date/Time Format"](https://xml2rfc.tools.ietf.org/public/rfc/html/rfc3339.html#anchor14), a manually check by ourselves will be the last choice if there isn't one.

Reply:
oh...sorry I just realized that I misunderstood your comment. Thought you meant there is a built-in js function that validates the date.

> I think a more accurate name for this function is isValidDate

@shimks `date-time` includes `date` and `time`, and this function is aimed to validate both, therefore IMO `isValidDateTime` is a more descriptive name.